lib/bundler/cli/gem.rb

```diff
@@ -124,6 +124,15 @@ def run
         templates.merge!("CODE_OF_CONDUCT.md.tt" => "CODE_OF_CONDUCT.md")
       end
 
+      if ask_and_set(:rubocop, "Do you want to add rubocop as a dependency for gems you generate?",
+        "RuboCop is a static code analyzer that has out-of-the-box rules for many " \
+        "of the guidelines in the community style guide. " \
+        "For more information, see the RuboCop docs (http://www.rubocop.org/en/latest/) " \
+        "and the Ruby Style Guides (https://github.com/rubocop-hq/ruby-style-guide).")
```

Comment:
The Ruby Style Guide URL has been updated to https://rubystyle.guide.

cf. https://github.com/rubocop-hq/rubocop/issues/7120